network/evpn.go

```diff
@@ -8,51 +8,113 @@ import (
 	"context"
 	"errors"
 	"log"
-
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
 	grpcOpi "github.com/opiproject/godpu/grpc"
-	pb "github.com/opiproject/opi-api/network/cloud/v1alpha1/gen/go"
+	pb "github.com/opiproject/opi-api/network/evpn-gw/v1alpha1/gen/go"
 	pc "github.com/opiproject/opi-api/network/opinetcommon/v1alpha1/gen/go"
 	"google.golang.org/grpc"
 	"google.golang.org/protobuf/types/known/emptypb"
+	"google.golang.org/protobuf/types/known/fieldmaskpb"
 )
-
-// PbEvpnClientGetter defines the function type used to retrieve an evpn protobuf client
-type PbEvpnClientGetter func(c grpc.ClientConnInterface) pb.CloudInfraServiceClient
+//PbEvpnBridgeClientGetter defines the function type used to retrieve an evpn protobuf LogicalBridgeServiceClient
+type PbEvpnBridgeClientGetter func(c grpc.ClientConnInterface) pb.LogicalBridgeServiceClient
+//PbEvpnPortClientGetter defines the function type used to retrieve an evpn protobuf BridgePortServiceClient
+type PbEvpnPortClientGetter func(c grpc.ClientConnInterface) pb.BridgePortServiceClient
```

Comment:
PbEvpnPortClientGetter -> PbEvpnBridgePortClientGetter

Reply:
In general update all the Bridge words to LogicalBridge
and all the Port words to BridgePort